### DISCUSSION TITLE
How to set ''crossref-resolve-refs' to false in quarto markdown.

### DISCUSSION THREAD

**Discussion by phongphuhanam** [quarto-dev/quarto-cli#4722] [Q&A]
Hello, I'm using quarto to create the PDF  with all the tables in LaTex format. I also need to create a docx version to share with my colleagues.  
As I want to cross-ref on both PDF and docx documents, I use the quarto crossref syntax (@tab-). It works fine with docx format, but for PDF I have to manually set ```param("crossref-resolve-refs", true)``` to ```param("crossref-resolve-refs", false)``` in this file  [ref.lua](https://github.com/quarto-dev/quarto-cli/blob/4c629a87596818b268ff424d663fd8b0ab3c5bb2/src/resources/filters/crossref/refs.lua#L27).
I want to set 'crossref-resolve-refs' to false for PDF output in quarto markdown but  I couldn't find any way to do so on the Quarto website.
Is there such an option exist? I tried the below setup but it did not work:
```yaml
format:
  pdf:
    crossref-resolve-refs : false
```

**Comment by cderv** [comment]
Sorry for missing that discussion. i'll close this in favor of #5161 that you created.
